### DISCUSSION TITLE
Visualizer Trame

### DISCUSSION THREAD

**Discussion by danielking00** [Kitware/trame#647] [Trame/ParaView]
Hey, I was wonder what the status of the visualizer application in Trame is. The last information I could find was from the github of the project; https://github.com/Kitware/paraview-visualizer/blob/master/README.rst, but the readme was last updated well over a year ago. So i was wandering if there has any developments. Am currently using ParaView Visualizer from web based visualization, but am condering moving over to trame when the visualizer tool is at the same stage as ParaView Visualizer

**Comment by jourdain** [comment]
Thanks for reaching out on that topic. The current trame version like mentioned is not complete and needs funding to be able to move forward. To ease any confusion, I just made that repository private (for now). 

If you would like to partner with Kitware to gain access or help improve it, you can reach out to [Kitware here](https://www.kitware.com/trame/).
